docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -39,74 +38,72 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
-> **NOTE**: If you installed the Intel® Distribution of OpenVINO™ to a non-default install directory, replace `C:\Program Files (x86)\Intel` with that directory in this guide's instructions.
+### <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
 
-You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables:
+You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables. Again, if you installed OpenVINO™ in a folder other than `C:\Program Files (x86)\Intel\openvino_2022`, use that location instead.
 
 ```sh
-"<INSTALL_DIR>\setupvars.bat"
+"C:\Program Files (x86)\Intel\openvino_2022\setupvars.bat"
 ```
 
-**Optional**: OpenVINO™ toolkit environment variables are removed when you close the command prompt window. You can permanently set the environment variables manually.
+> **Important**: The above command must be re-run every time a new Command Prompt window is opened.
 
-> **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. Check your system environment variables, and add Python if necessary.
+> **NOTE**: If you see an error indicating Python is not installed, Python may not be added to the PATH environment variable (as described [here](https://docs.python.org/3/using/windows.html#finding-the-python-executable)). Check your system environment variables, and add Python if necessary.
 
 The environment variables are set. Next, you can download some additional tools.
 
 ### <a name="model-optimizer">Step 3 (Optional): Install Additional Components</a>
 
-Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. The OpenVINO™ Development Tools can only be installed via PyPI now. See [Install OpenVINO™ Development Tools](installing-model-dev-tools.md) for detailed steps.
+OpenVINO Development Tools adds even more functionality to OpenVINO. It provides tools like Model Optimizer, Benchmark Tool, Post-Training Optimization Tool, and Open Model Zoo Downloader. If you install OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately. 
```

Comment:
This is only true for Python developers (who use CPU and GPU plugins). The plugin limitation is maybe out of scope for this document, but the Python part is important. 

For Python developers, should we be clearer upfront that PyPI already has everything they need?

Reply:
This one is confusing :( :( :( . Let me think for a bit.

Reply:
What if we change it to this?

"OpenVINO Development Tools is a set of utilities for working with OpenVINO and OpenVINO models. It provides tools like Model Optimizer, Benchmark Tool, Post-Training Optimization Tool, and Open Model Zoo Downloader. If you install OpenVINO Runtime using archive files, OpenVINO Development Tools must be installed separately.

See the [Install OpenVINO Development Tools](https://github.com/EdjeElectronics/openvino/blob/update-pypi-whats-next/docs/install_guides/installing-model-dev-tools.md) page for step-by-step installation instructions."

Reply:
Both suggestions are great, thanks @EdjeElectronics ! Unfortunately PyPI does not include MYRIAD/HDDL at the moment. Maybe to the second suggestion add something about that PyPI includes both CPU and GPU plugin?

Reply:
Okay, I changed the page to add both suggestions.